package.json

```diff
@@ -37,6 +37,7 @@
     "prepare": "husky install",
     "prepublishOnly": "npm run build",
     "prettier": "prettier .",
+    "mocha": "mocha",
```

Comment:
You can remove this. Users can add `./node_modules/.bin` to their PATH, or they can run `npx mocha`, both of which are easier than `npm run mocha`. It would be helpful if there were cli arguments, but for the default execution we don't need an npm script.

Reply:
Nice, forgot you could just use `npx`.